### ISSUE TITLE
Error running cx in blockchain broadcast mode

### ISSUE THREAD

**Issue by BigOokie** [skycoin/cx#357]
**Describe the bug**
Following the [CX Chains Tutorial Wiki](https://github.com/skycoin/cx/wiki/CX-Chains-Tutorial) I get the following error when I attempt to run CX in broadcast mode (`--braodcast`).

```bash
cx --broadcast --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
Blockchain counter: 1
panic: invalid character 'B' after top-level value

goroutine 1 [running, locked to thread]:
main.main()
	/go/src/github.com/skycoin/cx/cxgo/main.go:857 +0x549a
```

**To Reproduce**
1. Follow the steps outlined in the [CX Chains Wiki TL;DR section](https://github.com/skycoin/cx/wiki/CX-Chains-Tutorial)
2. Once you have both a `publisher` and a `peer` node running (in two seperate terminal sessions) run the test broadcast using the following command:
```bash
cx --transaction --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
```

This succeeds.

3. Next try to broadcast the transaction using the following command:
```bash
cx --broadcast --heap-initial 100 --stack-size 100 examples/blockchain/counter-txn.cx
```

This fails with the following error:
```bash
Blockchain counter: 1
panic: invalid character 'B' after top-level value

goroutine 1 [running, locked to thread]:
main.main()
	/go/src/github.com/skycoin/cx/cxgo/main.go:857 +0x549a
```

The error points to this block of code:
https://github.com/skycoin/cx/blob/4ba0d2d117e625622cc1a4224932ec4648602a49/cxgo/main.go#L855-L858

Debugging somewhat, I appear to be getting a `400 Bad Request` returned from this block of code:
https://github.com/skycoin/cx/blob/4ba0d2d117e625622cc1a4224932ec4648602a49/cxgo/main.go#L843-L847

**Expected behavior**
The transaction should be broadcast successfully onto the cxcoin blockchain.
Possibly some information is missing in the guide, or something is incorrectly configured.

**Screenshots**
NA. Refer to error information provided.

**Desktop (please complete the following information):**
 - OS: MacOS 10.14.4 (Mojave)
 - Browser: NA
 - CX Version: 0.6.2 - develop branch with blockchain integration. https://github.com/skycoin/cx/commit/4ba0d2d117e625622cc1a4224932ec4648602a49

**Additional context**
Revised version of the CX Chains Tutorial wiki was posted to the CXLabs Telegram group here:
https://t.me/CXLabs/856

**Comment by BigOokie** [comment]
I can confirm the API is accessable but running `skycoin-cli status` against the `cxcoin` node.

```bash
skycoin-cli status
```
```json
{
    "status": {
        "blockchain": {
            "head": {
                "seq": 1,
                "block_hash": "7d26d2b717e9ffebc1fc3753034b93e0490a09b16a23d2dd0d5c5a5cc6c80150",
                "previous_block_hash": "7a83bae77f02b7f22dc5026b437297e4db26fb0ab25203caaf5fceb419d9b2e8",
                "timestamp": 1557307182,
                "fee": 9999990000,
                "version": 0,
                "tx_body_hash": "7e3c5d4a7ca44d24ea084c66c1ee66e5de716d52356615781d2f50dcf774e322",
                "ux_hash": "91c8ac2c3325a56998be77f1c1a985e97996d369c282a856d88c148827268ab8"
            },
            "unspents": 1,
            "unconfirmed": 0,
            "time_since_last_block": "13h1m12s"
        },
        "version": {
            "version": "0.25.1",
            "commit": "",
            "branch": ""
        },
        "coin": "cxcoin",
        "user_agent": "cxcoin:0.25.1",
        "open_connections": 1,
        "outgoing_connections": 0,
        "incoming_connections": 1,
        "uptime": "20m56.261371667s",
        "csrf_enabled": true,
        "csp_enabled": true,
        "wallet_api_enabled": true,
        "gui_enabled": false,
        "unversioned_api_enabled": false,
        "json_rpc_enabled": false,
        "user_verify_transaction": {
            "burn_factor": 2,
            "max_transaction_size": 32768,
            "max_decimals": 3
        },
        "unconfirmed_verify_transaction": {
            "burn_factor": 2,
            "max_transaction_size": 200000,
            "max_decimals": 3
        },
        "started_at": 1557352798
    },
    "cli_config": {
        "webrpc_address": "http://127.0.0.1:6421"
    }
}
```

**Comment by ingwal** [comment]
What if you try with CX 0.7beta?  Does it still show the error?  Also, note that the instructions have changed somewhat, maybe we have fixed the issue in either of those.

**Comment by BigOokie** [comment]
Pretty sure this was resolved by the 0.7beta release. I will do some further testing (hopefully this weekend) and confirm
